src/lib.rs

```diff
@@ -161,6 +161,8 @@ pub struct Args {
     /// Specify the VCS used to initialize the generated template.
     #[structopt(long, default_value = "git")]
     pub vcs: Vcs,
+    #[structopt(long = "bare")]
```

Comment:
I find the name of this argument a bit confusing / misleading.
Git has a `--bare` option that initialises a bare repo, one that can be used as a `remote` directory for push.

Also I'm missing the doc blocks above this flag, that would explain the purpose of this flag to a user (when calling `--help`)

Reply:
maybe `--skip-git-init` would express the purpose a bit better?

Reply:
Good remark about the missing doc block. I just added it.

The purpose is not to skip git init. That has been dealt with by other issues (see https://github.com/cargo-generate/cargo-generate/issues/193#issuecomment-779390079). Rather, the purpose is to be able to avoid creating a directory for the project. It's the same difference between `cargo new` and `cargo init`.

From the beginning, I haven't been sure as to what to call the option (see https://github.com/cargo-generate/cargo-generate/issues/193#issuecomment-537634814 to which there was never a response). The inspiration for `--bare` comes from Haskell Stack (see https://github.com/commercialhaskell/stack/issues/965). Just like you, I wasn't entirely convinced because of the possible confusion with `git init --bare`.

Looking around and thinking about it a bit more, the idea of `--skip-directory` appeals to me. @sassman, what do you think?

What do others think? I am more than open to getting more opinions, as long as we avoid bikeshedding.

Reply:
thanks for pointing this out. Now I know why I got confused: `cargo init fo` would actually also create a folder `fo` in `$CWD`. But when the last argument `fo` would be omitted it initializes in `$CWD`. 

So this would actually than mean something like `--into-current-dir` or so. I find the connotation of `skip` then rather misleading. Because it will not skip anything but rather it will create something right in here, in this current working directory.

Reply:
How about `use-current-dir`?